src/Http/Http.Abstractions/src/EndpointFilterInvocationContextOfT.Generated.tt

```diff
@@ -52,8 +52,8 @@ internal sealed class EndpointFilterInvocationContext<<# foreach (var argumentCo
 
     public override IList<object?> Arguments => this;
 
-    <# foreach (var argumentCount in Enumerable.Range(0, arity)) { #>public T<#=argumentCount#> Arg<#=argumentCount#> { get; set; }
-    <# } #>
+<# foreach (var argumentCount in Enumerable.Range(0, arity)) { #>    public T<#=argumentCount#> Arg<#=argumentCount#> { get; set; }
+<# } #>
```

Comment:
Just to explain this change, I'm on macOS and using either Rider or VSCode the editors immediately delete the whitespace from the generated files the moment I touch them, so this change makes editing them by hand less volatile.

Why am I editing the generated code by hand? Because when I try to use the tt generator it completely re-writes the file in a way that creates a massive diff (the header text is different for example).

Reply:
Might be worth at some point having the repo updated to use the [.NET 6 version](https://devblogs.microsoft.com/dotnet/t4-command-line-tool-for-dotnet/) of the tt generator - maybe that'll remove some of the xplat wonk (after an initial normalisation).

Reply:
#49382